container/container.go

```diff
@@ -282,10 +289,17 @@ func PortBindingsForTask(taskInfo *mesos.TaskInfo) map[docker.Port][]docker.Port
 		if port.HostPort == nil {
 			continue
 		}
-		portBinds[docker.Port(strconv.Itoa(int(*port.ContainerPort))+"/tcp")] = // TODO UDP support?
-			[]docker.PortBinding{
-				docker.PortBinding{HostPort: strconv.Itoa(int(*port.HostPort))},
-			}
+
+		protocols := port.GetProtocol()
```

Comment:
This should be broken out into a function since we're doing the exact same thing here.